lld/ELF/Config.h

```diff
@@ -473,6 +473,7 @@ struct Ctx {
                  std::pair<const InputFile *, const InputFile *>>
       backwardReferences;
   llvm::SmallSet<llvm::StringRef, 0> auxiliaryFiles;
+  InputFile *internalFile;
```

Comment:
Will be worth leaving a comment here, perhaps:
```
InputFile for linker created symbols with no source location.
```

Reply:
Thanks for the suggestion!